packages/autop/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## Unreleased (1.1.0)
```

Comment:
I think formatting would be `## 1.1.0 (Unreleased)` for consistency with other releases?

Reply:
I can update all of them 👍